src/vtrasterizer/BoxDrawingRenderer.cpp

```diff
@@ -1954,6 +1954,7 @@ auto buildBox(detail::Box box, ImageSize size, int lineThickness, size_t supersa
             case Heavy4: return false;
         }
         assert(false);
+        return false;
```

Comment:
p.s.: actually, I think we should use:


```suggestion
        std::unreachable();
```
instead.

https://en.cppreference.com/w/cpp/utility/unreachable.html